css/src/atomics/width.scss

```diff
@@ -22,6 +26,10 @@ $all-widths: list.join($universal-widths, $mobile-incompatible-widths);
 		width: 100% !important;
 	}
 
+	.width-fit-content-tablet {
+		width: fit-content;
+	}
+
 	@each $width in $all-widths {
 		$unit: if($width != auto and $width != unset, 'px', '');
```

Comment:
Thoughts on this?

if we do this:
`		$unit: if($width != auto and $width != unset and $width != fit-content, 'px', '');` then we can just add `fit-content` to the array of values at the top of the file (`$universalwidths`).

Reply:
Definitely more efficient😎, updated 👍🏾; exceptions covered by the if is getting lengthy🫣